modules/contracts/apps/calendar/src/contract.rs

```diff
@@ -13,20 +13,21 @@ pub const APP_VERSION: &str = env!("CARGO_PKG_VERSION");
 pub const APP_ID: &str = "abstract:calendar";
 
 /// The type of the result returned by your app's entry points.
-pub type AppResult<T = Response> = Result<T, AppError>;
+pub type CalendarAppResult<T = Response> = Result<T, AppError>;
```

Comment:
Shouldn't it be CalendarError then?
```suggestion
pub type CalendarAppResult<T = Response> = Result<T, CalendarError>;
```

Reply:
Sure, will also change that.